src/components/CategoryHeading.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+  import { areAllDefined } from "../util/genUtil";
+  import { renderPercentage } from "../util/numberUtil";
+  import { renderCatHeadString } from "../helpers/categoryHelper"; 
+  export let variableData
+  export let variable
+  export let category
+  export let location;
+</script>
+
+<h2>
```

Comment:
Don't think this is an `h2` - it would be the top level heading in the design (so `h1`) but I don't think it's a correct use of an `h` element to put [block-level content inside it](https://stackoverflow.com/a/19779520). Maybe @designorant could help?

Reply:
happy to go with whatever, but in the designs it does look like this thing is a heading (should also rename the component if it isn't)

Reply:
changed to plain div, pending style pass